src/getCommands.js

```diff
@@ -1,27 +1,17 @@
 const path = require('path');
-const flatten = require('lodash.flatten');
+const fs = require('fs');
 const uniq = require('lodash.uniq');
+const flatten = require('lodash.flatten');
 const findPlugins = require('./findPlugins');
 
 /**
- * Get plugin commands
- * @param  {String}       name Name of the plugin
- * @return {Array|Object}      Plugin's commands
- */
-const getPluginCommands = cwd => name =>
-  require(path.join(cwd, 'node_modules', name));
-
-/**
  * @return {Array} Array of commands
  */
 module.exports = function getCommands() {
   const rnpmRoot = path.join(__dirname, '..');
   const appRoot = process.cwd();
 
-  const pluginsList = flatten([
-    findPlugins(rnpmRoot).map(getPluginCommands(rnpmRoot)),
-    findPlugins(appRoot).map(getPluginCommands(appRoot)),
-  ], true);
-
-  return uniq(pluginsList, 'name');
+  return uniq(
+    flatten(findPlugins([rnpmRoot, appRoot]).map(require))
```

Comment:
That still does not work, as per my comment on outdated diff. It will work in ALL CASES except when rnpm is installed globally and app plugins are requested, because then -> require will look in rnpm own modules and in global node modules missing appRoot entirely. So as I said -> we should use `require(path.join(cwd, 'node_modules', name))` for appRoot plugins only (that's super easy and explicit) and leave `require` magic for `rnpmRoot` entirely :)